Makefile

```diff
@@ -29,9 +38,23 @@ req:
 install: prepare
 	$(PREFIX)pip install .
 
+# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
 install-sdk:
-	# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
+ifeq ($(PLATFORM), darwin)
 	which atlas-run-standalone || brew tap atlassian/tap && brew install atlassian/tap/atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/debian_version),)
+	sudo sh -c 'echo "deb https://sdkrepo.atlassian.com/debian/ stable contrib" >/etc/apt/sources.list.d/atlassian_development.list'
```

Comment:
If this fails do we call back to informing the user that the sdk cannot be installed ? (for example in the case of a user not having sudo access)

Reply:
I'm doing a check to see if the user is root (sudo will also show them as root)